avro-fastserde/src/jmh/java/com/linkedin/avro/fastserde/fastAvroSerdesBenchmark.java

```diff
@@ -0,0 +1,110 @@
+package com.linkedin.avro.fastserde;
+
+import com.linkedin.avro.fastserde.generated.avro.BenchmarkSchema;
+import com.linkedin.avro.fastserde.generator.AvroRandomDataGenerator;
+import com.linkedin.avro.fastserde.micro.benchmark.AvroGenericDeserializer;
+import com.linkedin.avro.fastserde.micro.benchmark.AvroGenericSerializer;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Random;
+import java.util.concurrent.TimeUnit;
+import org.apache.avro.Schema;
+import org.apache.avro.generic.GenericData;
+import org.apache.avro.generic.GenericDatumReader;
+import org.apache.avro.generic.GenericRecord;
+import org.openjdk.jmh.annotations.Benchmark;
+import org.openjdk.jmh.annotations.BenchmarkMode;
+import org.openjdk.jmh.annotations.Fork;
+import org.openjdk.jmh.annotations.Level;
+import org.openjdk.jmh.annotations.Measurement;
+import org.openjdk.jmh.annotations.Mode;
+import org.openjdk.jmh.annotations.OutputTimeUnit;
+import org.openjdk.jmh.annotations.Scope;
+import org.openjdk.jmh.annotations.Setup;
+import org.openjdk.jmh.annotations.State;
+import org.openjdk.jmh.annotations.Warmup;
+
+
+/**
+ * A benchmark that evaluates the performance of serialize and deserialize of the given Avro Schema and
+ * parameters using original Apache Avro GenericDatumReader/Writer and FastGenericDatumReader/Writer in
+ * different Avro versions
+ *
+ * To run this benchmark:
+ * <code>
+ *   ./gradlew :avro-fastserde:jmh -PUSE_AVRO_14/17/18
+ * </code>
+ *
+ * You also can test by your own AVRO schema by replacing the contents in
+ *   avro-util/avro-fastserde/src/test/avro/benchmarkSchema.avsc
+ */
+@State(Scope.Benchmark)
+@BenchmarkMode(Mode.AverageTime)
+@OutputTimeUnit(TimeUnit.NANOSECONDS)
+@Fork(1)
+//@Fork(value = 1, jvmArgsAppend = {"-XX:+PrintGCDetails", "-Xms16g", "-Xmx16g"})
+//@Threads(10)
+@Warmup(iterations = 5)
+@Measurement(iterations = 5)
+public class fastAvroSerdesBenchmark {
```

Comment:
To be consistent, can we use name: `FastAvroSerdesBenchmark`?

Reply:
Fixed